battleship-server/src/main/java/com/epam/solid/nie/config/FileConfiguration.java

```diff
@@ -7,12 +7,12 @@
 import java.util.stream.Stream;
 
 public class FileConfiguration implements Configuration {
-
+    public static final String CONFIGURATION = "configuration";
```

Comment:
That is supposed to be config file name, right? If so, let's name it CONFIG_FILENAME perhaps?